skyvern/services/script_service.py

```
@@ -686,6 +691,162 @@ async def _run_cached_function(cached_fn: Callable) -> Any:
+    """
+    context = skyvern_context.current()
+    if not context or not context.organization_id or not context.script_revision_id:
+        return
+
+    try:
+        script_block = await app.DATABASE.get_script_block_by_label(
+            organization_id=context.organization_id,
+            script_revision_id=context.script_revision_id,
+            script_block_label=cache_key,
+        )
+    except Exception:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)